pkgs/by-name/eb/ebpf-usb/package.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, bcc
+, makeWrapper
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "ebpf-usb";
+  version = "unstable-2022-04-03";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "francisrstokes";
+    repo = "ebpf-usb";
+    rev = "3ab6f0d8c6ece51bbb5cc5e05daa4008eccd70e8";
+    sha256 = "n3ttFej9sroTqAOgyAejwKT+aMt/z7HlVPV6CVGPNUQ=";
```

```suggestion
    hash = "sha256-n3ttFej9sroTqAOgyAejwKT+aMt/z7HlVPV6CVGPNUQ=";
```
sri hashes are prefered